packages/hemi-tunnel-actions/index.ts

```diff
@@ -0,0 +1,2 @@
+export { depositErc20Token } from './src/depositErc20Token'
```

Comment:
All ERC-20 are tokens so `depositErc20Token` could easily be `depositErc20` with the exact same meaning but shorter. Just a note in the case we feel like shortening the names.

Reply:
Renamed here [e4ee98a](https://github.com/hemilabs/ui-monorepo/pull/1119/commits/e4ee98a05f09e0c6a9657388dedb2559b7bafc6c)